activerecord/lib/active_record/enum.rb

```diff
@@ -4,7 +4,7 @@
 require "active_support/core_ext/object/deep_dup"
 
 module ActiveRecord
-  # Declare an enum attribute where the values map to integers in the database,
+  # Declare an enum attribute where the values map to integers/strings in the database,
```

Comment:
We should, probably, update these docs to describe better the usage with string columns. I'm waiting @kaspth 's confirmation to tackle this.